test/units/res.cookie.test.ts

```diff
@@ -17,6 +17,32 @@ describe("res", function () {
         .expect(200, done);
     });
 
+    it("should raise an exception if no arg is provided", function (done) {
+      const app = opine();
+
+      app.use(function (req, res) {
+        (res as any).cookie().end();
```

Comment:
Assuming the typing is working correctly I would hope this `any` casting shoudn't be required (as per previous tests).

```suggestion
        res.cookie().end();
```